scripts/run_lint.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+find . -name '*.py' | xargs pylint
```

Comment:
I couldn't find a way of doing this without a separate script, `pyproject.toml` seems to be a bit retarded when it comes to running bash instructions.

Also, with the changes added in #1292, the CLI output was very polluted by poetry which printed out all affected files (they make a pretty big list).